src/wx/wxvbam.h

```diff
@@ -382,7 +382,8 @@ class MainFrame : public wxFrame {
     // Load a named wxDialog from the XRC file
     wxDialog* LoadXRCropertySheetDialog(const char* name);
 
-    wxChoice* pixel_filters = nullptr;
+    wxChoice* pixel_filters       = nullptr;
+    wxChoice* interframe_blenders = nullptr;
```

Comment:
I'd suggest `interframe_blenders_` with a trailing `_`. It makes it easier to see we are using a private member here and not a variable. I know this is a style change from the rest of the file but I feel like it really helps readability and maintainability.